docs/index.md

```diff
@@ -55,59 +55,6 @@ Trivy is integrated with many popular platforms and applications. The complete l
 - [VS Code plugin](https://github.com/aquasecurity/trivy-vscode-extension)
 - See [Ecosystem] for more
 
-### General usage
```

Comment:
I think this page should still give some kind of impression on how using the tool is like. not a complete walkthrough just an example.